src/Core/Core/Utils/Coordinate.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace QuantumCore.Core.Utils;
+
+public record Coordinate(int X, int Y);
```

Comment:
I think u can use the built in `Point` type from `System.Drawing`. The name isn't as fitting but on the other side its much more optimized